braid/base.h

```diff
@@ -78,6 +89,17 @@ _braid_BaseClone(braid_Core         core,     /**< braid_Core structure */
                  braid_BaseVector   u,        /**< vector to clone */ 
                  braid_BaseVector  *v_ptr     /**< output, newly allocated and cloned vector */ 
                  );
+
+/**
+ * This initializes a braid_Basis and calls the user's clone routine on each column.
```

Comment:
Maybe say
"This initializes a braid_Basis by cloning A into B, using the user's clone routine"
?